src/test/kotlin/in/rcard/assertj/arrowcore/EitherAssert_hasLeftValueSatisfying_Test.kt

```diff
@@ -0,0 +1,42 @@
+package `in`.rcard.assertj.arrowcore
+
+import arrow.core.Either
+import `in`.rcard.assertj.arrowcore.errors.EitherShouldBeLeft
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.util.FailureMessages.actualIsNull
+import org.junit.jupiter.api.Test
+
+
+class EitherAssert_hasLeftValueSatisfying_Test {
+
+    @Test
+    internal fun `should fail when either is null`() {
+        val actual: Either<Int, String>? = null
+        Assertions.assertThatThrownBy { EitherAssert(actual).hasLeftValueSatisfying { } }
```

Comment:
Don't use the directly the `EitherAssert(actual)` constructor. Use the `EitherAssert.assertThat(actual)` factory method instead.

This comment applies everywhere you have used the constructor.